UndertaleModTool/Windows/SearchInCodeWindow.xaml.cs

```diff
@@ -53,9 +53,20 @@ public partial class SearchInCodeWindow : Window
 
         static bool isSearchInProgress = false;
 
-        public SearchInCodeWindow()
+        public SearchInCodeWindow(string query = null, bool inAssembly = false)
         {
             InitializeComponent();
+
+            if (query is not null)
+            {
+                if (query.Length > 256 || query.Count(x => x == '\n') > 16)
+                    return; // Ignore if the query is longer than 256 characters or 16 lines.
+
+                SearchTextBox.Text = query;
+                SearchTextBox.SelectAll();
+            }
+
+            InAssemblyCheckBox.IsChecked = inAssembly;
```

Comment:
Very small logic thing, but I think the `return` from above will accidentally skip this if you're selecting a lot of text in diassembly.

Reply:
Hm, but if the length is too big, then it shouldn't check the checkbox as it counts as you have selected nothing in disassembly?

Reply:
I would think the user *intended* to search in the disassembly if that's the standard selection behavior... but not sure. Your call, I guess. I don't care too much.